notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_inpainting.ipynb

```diff
@@ -0,0 +1,625 @@
+{
```

Comment:
<p>use default for REGION -&gt; us-central1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
done